app/routes.js

```diff
@@ -62,6 +62,21 @@ module.exports = function(app) {
             .exec(resHandler(res))
     });
 
+    // get all neighbours
+    app.get('/api/sample/neighbours/:id', function(req, res) {
```

Comment:
The next PR (faster loading) is going to include quite a few changes to the DB API -- simplying the routes and making them more general. I'll address these comments in that PR.